File: Amigo.Tenant.Application.DTOs/Responses/Tracking/EquipmentSizeDTO.cs

namespace Amigo.Tenant.Application.DTOs.Responses.Tracking
{
  public  class EquipmentSizeDTO: IEntity
    {
        public int EquipmentSizeId { get; set; }
        public string Name { get; set; }
        public string Code { get; set; }
        public int?  EquipmentTypeId { get; set; }
        public string EquipmentTypeCode { get; set; }
        public bool RowStatus { get; set; }

    }
}

File: Amigo.Tenant.UI.Web/src/app/shared/dtos/equipmentSizeDTO.ts
import {Component} from '@angular/core';

export class EquipmentSizeDTO {

    equipmentSizeId: number;
    equipmentTypeId: number;
    equipmentTypeCode: string;
    name: string;
    code: string;
    rowStatus: string;

    constructor(private equipmentSizeId1: number, private name1: string, code1: string) {
        this.equipmentSizeId = equipmentSizeId1;
        this.name = name1;
        this.code = code1;
    }


}